src/com/farzaneh/filter/SecurityFilter.java

```diff
@@ -0,0 +1,48 @@
+package com.farzaneh.filter;
+
+import java.io.IOException;
+
+import javax.servlet.Filter;
+import javax.servlet.FilterChain;
+import javax.servlet.FilterConfig;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRequest;
+import javax.servlet.ServletResponse;
+import javax.servlet.annotation.WebFilter;
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+
+import com.farzaneh.service.dao.PersonDAO;
+
+@WebFilter(filterName = "/SecurityFilter", urlPatterns = "/shoppingCartServlet")
+public class SecurityFilter implements Filter {
+
+	public SecurityFilter() {
+	}
+
+	public void destroy() {
+	}
+
+	public void doFilter(ServletRequest request, ServletResponse response, FilterChain chain)
+			throws IOException, ServletException {
+		HttpServletRequest req = (HttpServletRequest) request;
+		for (Cookie cookie : req.getCookies()) {
```

Comment:
instead of handling security with Cookies try to use HttpSession

Reply:
done